docs_user/modules/proc_rolling-back-the-control-plane-adoption.adoc

```diff
@@ -138,26 +130,23 @@ for i in {1..3}; do
 done
 ----
 
-. If the Ceph NFS service is running on the deployment as a {rhos_component_storage_file_first_ref}
-backend, you must restore the pacemaker ordering and colocation constraints
-involving the "openstack-manila-share" service:
+. If the Ceph NFS service is running on the deployment as a {rhos_component_storage_file_first_ref} back end, you must restore the Pacemaker order and colocation constraints for the `openstack-manila-share` service:
 +
 ----
 
-sudo pcs constraint order start ceph-nfs then openstack-manila-share kind=Optional id=order-ceph-nfs-openstack-manila-share-Optional
-sudo pcs constraint colocation add openstack-manila-share with ceph-nfs score=INFINITY id=colocation-openstack-manila-share-ceph-nfs-INFINITY
+$ sudo pcs constraint order start ceph-nfs then openstack-manila-share kind=Optional id=order-ceph-nfs-openstack-manila-share-Optional
+$ sudo pcs constraint colocation add openstack-manila-share with ceph-nfs score=INFINITY id=colocation-openstack-manila-share-ceph-nfs-INFINITY
 
 ----
 
-. Verify that the source cloud is operational again, e.g. by
-running `openstack` CLI commands or using the {dashboard_first_ref}.
-//kgilliga: Should we be more specific? Which CLI commands?
+. Verify that the source cloud is operational again, for example, you
+can run `openstack` CLI commands or check that you can access the {dashboard_first_ref}.
```

Comment:
1. Are we saying that they can run any openstack command, and if it’s successful, the cloud is running again? Similarly, if they can access the Dashboard, the cloud is operational?
2. What would the customer need to do if the source cloud is not operational?

Reply:
1. It doesn't guarantee that all parts of the cloud are working ok, but it's a good smoke test. Really validating full functionality is only possible by something like running the Tempest test suite against the cloud, and we probably shouldn't be recommending that. So it's not a comprehensive validation but i don't think it's worth to aiming for one.
2. Start troubleshooting. Potentially it may lead to "restore from backup" situation. But i think it's very likely that the rollback would succeed, or require just minor tweaks after troubleshooting, without needing to restore from backup.